frontend/src/components/commons/SideNavigation.jsx

```diff
@@ -11,8 +11,39 @@ class SideNavigation extends Component {
     currentNode: this.props.currentNode,
     currentBody: this.props.progressSpecs[this.props.currentNode].body
   };
+
+  changeNode(id) {
+    this.setState({ currentNode: id, currentBody: this.props.progressSpecs[id].body });
+  }
+
   render() {
-    return <div>STUBBED SideNavigation Component</div>;
+    return (
+      <div className="side-nav-pane">
+        <div className="side-nav-btn-list">
+          {this.props.progressSpecs.map(tab => (
```

Comment:
I'm confused by the property naming. What does `progressSpecs` mean?

Reply:
renamed in e83657b